docs/guides/Orders/orders-feed.md

```diff
@@ -358,33 +324,28 @@ When the hook is configured, VTEX sends a ping to the endpoint given in the conf
 ```
 
 [block:callout]
-{
-  "type": "warning",
-  "body": "The given endpoint should return a status 200 for the above-mentioned request. Otherwise, the Hook API will return a status `400 Bad Request`, and saving the configuration won't be possible."
-}
-[/block]
+{ "type": "warning", "body": "The given endpoint should return status 200 for the above-mentioned request. Otherwise, the Hook API will return status `400 Bad Request`, and you won't be able to save the configuration." } [/block]
```

Comment:
```suggestion
>⚠️ The given endpoint should return status 200 for the above-mentioned request. Otherwise, the Hook API will return status `400 Bad Request`, and you won't be able to save the configuration.
```